### ISSUE TITLE
Unable to login to admin

### ISSUE THREAD

**Issue by CallanC** [h9k/magirc#130]
Hey h9k, 

I am running into a problem. 

To start, here is the following information:
magirc: latest git (June 10, 2016)
nginx: nginx/1.10.1
mysql: mysql  Ver 14.14 Distrib 5.5.49, for debian-linux-gnu (x86_64) using readline 6.3
php: PHP 7.0.7-1~dotdeb+8.1 (cli) ( NTS )

If you need to look at the config files for nginx on how it reads the conf, let me know.

The login page displays, but I am unable to login to proceed further. The /setup errors with the following:

`Fatal error: Uncaught Error: Call to undefined method Twig_Environment::assign() in /var/www/magirc/setup/lib/Setup.class.php:119 Stack trace: #0 /var/www/magirc/setup/inc/step2.php(6): Setup->saveConfig() #1 /var/www/magirc/setup/index.php(39): include('/var/www/magirc...') #2 {main} thrown in /var/www/magirc/setup/lib/Setup.class.php on line 119`

If I manually create and modify **conf/magirc.cfg.php**, the login page displays only after informing me that I need to re-run /setup to configure the SQL Config Table. Once this has been done, it prompts the login page, I enter the credentials and it seems to just refresh the page. 

Reading from the closed issues, to note: 

In phpMyAdmin, the magirc database, magirc_config, has a base URL with a trailing slash (i.e. http://stats.panicirc.net/). I have attempted multiple variations to see if it was something on my end, such as downloading magirc from http://www.magirc.org/, composer, and git, with all of them coming to the same problem. 

Additionally, no errors are being posted to the log file other than the errors described above for Twig.

During the initial setup, magirc does not inform me of anything missing, all show Present and Writable. 

Any ideas?

**Comment by h9k** [comment]
Maybe you have something broken in the vendor libs... try to delete the content of the vendor folder and also delete the composer.lock file and run composer install again.
Or maybe there is some issue with magirc and PHP 7.0, I haven't tested it yet.

**Comment by CallanC** [comment]
I reverted back to php5 and it seems to have the same problem.

```
Configuration File (php.ini) Path	/etc/php5/fpm
Loaded Configuration File	/etc/php5/fpm/php.ini
Scan this dir for additional .ini files	/etc/php5/fpm/conf.d
```
Gonna make a new VM and test it out and see what happens. 

I removed all the magirc files and started fresh again and noticed during the initial setup, there were two 'database' fields. Don't know if this is something with my setup, or something that gets generated after running composer. 

Otherwise, I'll keep an eye on any new updates if you begin playing with php7.0.

Thank you!
